### DISCUSSION TITLE
Failed to download laravel/nova from dist: Invalid credentials for

### DISCUSSION THREAD

**Discussion by rmeira** [laravel/nova-issues#3985] [General]
- Laravel Version: 8.0
- Nova Version: 3.0
- PHP Version: 7.4
- Database Driver & Version:
- Operating System and Version: Mac OS 13.3M1
- Browser type and version: #.#
- Reproduction Repository: https://github.com/###/###

### Description:

Hi Guys, help me!
I Cant install laravel nova, see the messega bellow

<img width="951" alt="image" src="https://user-images.githubusercontent.com/8323576/163248629-e795b553-5c21-4471-abc8-ed1e484d78eb.png">

My composer.json

```
{
    "name": "laravel/laravel",
    "type": "project",
    "description": "The Laravel Framework.",
    "keywords": [
        "framework",
        "laravel"
    ],
    "license": "MIT",
    "require": {
        "php": "^7.4",
        "artisaninweb/laravel-soap": "0.3.0.9",
        "beberlei/assert": "^3.2",
        "bissolli/nova-phone-field": "^0.0.1",
        "bugsnag/bugsnag-laravel": "^2.0",
        "chaseconey/nova-external-image": "^0.0.3",
        "digital-creative/collapsible-resource-manager": "^1.1",
        "dillingham/nova-button": "^1.0",
        "doctrine/dbal": "^2.9",
        "facade/ignition": "^2.3.6",
        "fideloper/proxy": "^4.0",
        "fruitcake/laravel-cors": "^2.0",
        "guzzlehttp/guzzle": "^7.0.1",
        "insenseanalytics/nova-bar-metrics": "^1.0",
        "laravel/framework": "^8.0",
        "laravel/helpers": "^1.2",
        "laravel/nova": "~3.0",
        "laravel/passport": "^10.0",
        "laravel/tinker": "^2.0",
        "laravel/ui": "^3.0",
        "lcobucci/jwt": "3.3.3",
        "lhilton/text-auto-complete": "^1.0",
        "lucascudo/laravel-pt-br-localization": "^1.0",
        "maatwebsite/excel": "^3.1",
        "mattsplat/readmore": "^1.0",
        "media24si/nova-youtube-field": "^1.0",
        "morrislaptop/laravel-queue-clear": "~1.0",
        "multilaser/back-button": "*",
        "multilaser/import-blacklist": "*",
        "multilaser/import-caids": "*",
        "multilaser/multiplikpro-xs": "*",
        "multilaser/search-code": "*",
        "multilaser/support-chat": "*",
        "multilaser/wallet-extract": "*",
        "orlyapps/nova-belongsto-depend": "^2.0",
        "pdewit/nova-external-url": "^1.0",
        "pragmarx/version": "^1.0",
        "predis/predis": "^1.1",
        "rap2hpoutre/laravel-log-viewer": "^1.3",
        "rebing/graphql-laravel": "^5.1",
        "sixlive/nova-text-copy-field": "^1.5",
        "spatie/enum": "^2.3",
        "spatie/laravel-ray": "^1.11",
        "spatie/laravel-sluggable": "^2.1",
        "spatie/simple-excel": "^1.13",
        "stonkeep/cpf-cnpj": "^0.0.1",
        "stonkeep/moeda-real": "dev-master",
        "timothyasp/nova-badge-field": "^1.0",
        "timothyasp/nova-color-field": "^1.0",
        "titasgailius/search-relations": "^1.0",
        "vyuldashev/nova-money-field": "^1.3",
        "vyuldashev/nova-permission": "^2.4",
        "wemersonrv/input-mask": "^0.1.2",
        "multilaser/import-oi-services": "*",
        "multilaser/import-claro-payment": "*"
    },
    "require-dev": {
        "barryvdh/laravel-debugbar": "^3.2",
        "beyondcode/laravel-dump-server": "^1.0",
        "beyondcode/tinkerwell-helper": "^1.0",
        "filp/whoops": "^2.0",
        "fzaninotto/faker": "^1.4",
        "marvinrabe/laravel-graphql-test": "^0.2.0",
        "mockery/mockery": "^1.0",
        "nunomaduro/collision": "^5.0",
        "pestphp/pest": "^1.0",
        "pestphp/pest-plugin-faker": "^1.0",
        "pestphp/pest-plugin-laravel": "^1.0",
        "phpunit/phpunit": "^9.3.10"
    },
    "config": {
        "optimize-autoloader": true,
        "preferred-install": "dist",
        "sort-packages": true
    },
    "extra": {
        "laravel": {
            "dont-discover": []
        }
    },
    "autoload": {
        "psr-4": {
            "App\\": "app/",
            "Library\\": "app/Library",
            "Database\\Factories\\": "database/factories/",
            "Database\\Seeders\\": "database/seeders/"
        },
        "files": [],
        "classmap": []
    },
    "autoload-dev": {
        "psr-4": {
            "Tests\\": "tests/"
        }
    },
    "minimum-stability": "dev",
    "prefer-stable": true,
    "repositories": [
        {
            "type": "composer",
            "url": "https://nova.laravel.com"
        }
    ]
}
```

**Comment by misenhower** [comment]
We just ran into this as well. The solution was to change the password in `auth.json` to the Nova license key instead of the user account's password.

Go to https://nova.laravel.com/licenses, click your license, and you should see the License Key listed on the details page. Hope this helps!

**Edit:** Sounds like a lot of people are still having issues even with the license key 😕  I'm not sure why it fixed it for us then, sorry!

**Comment by ibrunotome** [comment]
I'm facing this too today. I always used license key instead account password, but this still happening.

**Comment by john-climer** [comment]
Same issue here. We've been using the license key. I tried rotating the key but getting the same invalid credentials error. We are on v3.30 if it helps.

**Comment by mattpatera-ych** [comment]
We are also having this issue. I also tried to look up the license key for our license using the link above and we don't have a key, just username and password.

![image](https://user-images.githubusercontent.com/47829321/163262920-bfae83b1-cc2f-461e-aff4-5247bee8335b.png)

**Comment by wilburpowery** [comment]
Started seeing the following issue today too: 
![Screen Shot 2022-04-13 at 2 28 43 PM](https://user-images.githubusercontent.com/15817188/163264997-894a881a-f581-4839-885b-93d019537d11.png)

It just started happening today too, worked fine up to yesterday.

**Comment by kfrymire** [comment]
I'm having this issue too.  Nova suddenly stopped installing and our CI builds are failing as a result.  I tried changing the auth.json password to license key, changed the nova password, set nova version in composer to ~3.30 (we can't upgrade to 4 yet).  Nothing worked. Help us Obi-wan... we cannot update our app.

<img width="1385" alt="Screen Shot 2022-04-13 at 3 45 03 PM" src="https://user-images.githubusercontent.com/16312945/163267650-e662363c-b7b1-48e9-b116-3819126bad01.png">

**Comment by isajar** [comment]
Same problem here!!

**Comment by ImJustToNy** [comment]
We've been using license key whole time and it stopped working right now.

**Comment by rmeira** [comment]
> We just ran into this as well. The solution was to change the password in `auth.json` to the Nova license key instead of the user account's password.
> 
> Go to https://nova.laravel.com/licenses, click your license, and you should see the License Key listed on the details page. Hope this helps!

I try to change password at https://nova.laravel.com/settings/password bot not resolve.

<img width="1637" alt="image" src="https://user-images.githubusercontent.com/8323576/163272918-27769372-4339-4df6-a59b-b7791685db03.png">

**Comment by pitylee** [comment]
Same here.

**Comment by nshontz** [comment]
Same same.

**Comment by kfrymire** [comment]
This is a serious blocker.  Composer seems to be forcing Nova 4 (Silver Surfer) even if I specify Nova 3 (Orion).  We're stuck on Laravel 7 for now, which means we can't upgrade to Laravel 8 or 9 which Nova 4 requires (I can't upgrade even if I wanted to).  Nova 3.31 dropped Laravel 7 support, but I can't install Nova 3 so our project is dead in the water.  I can't even work locally on it.

**Comment by pitylee** [comment]
Until now, I had a cached version of 3.30.0, which was a life saviour, for a few days it was okay, now, not even that works, and I am locked to ~3.0, which should exclude v4 for you too @kfrymire.

This needs to be solved asap, it is pretty serious, think about all the projects that can, or did go wrong right now. (it is happening for a few hours)

I think they handed out free upgrades to v4 for those that had v3 Pro, and there's some trouble with their routing.

**Comment by crynobone** [comment]
Hi all, the issue should be fixed now. Sorry for the inconvenience.

![image](https://user-images.githubusercontent.com/172966/163276431-e110a2ac-74b9-4e08-92a2-b824a5b53bef.png)

**Reply by kfrymire** [reply]
Huzzah! That seems to have done the trick - Nova 3.30 is working for me again.  thanks!

**Reply by rmeira** [reply]
I test now and working for me too, thanks!
